packages/libs/client/src/pact.ts

```diff
@@ -220,7 +220,8 @@ export class PactCommand
     // convert to IUnsignedTransaction
     const command: ICommand = {
       hash,
-      sigs: this.sigs.map((s) => (s === undefined ? { sig: '' } : s)),
+      // eslint-disable-next-line @rushstack/no-new-null
+      sigs: this.sigs.map((s) => ({ sig: s?.sig ?? null })),
```

Comment:
Should this `null` value be dealt with in `local()` function

Reply:
`sigs` are signed before sent to `/local`, but now we take in invalid sig strings when `signatureVerification` is turned off. We can let sig be undefined here, but we need to create new type instead of `ICommand`. Let's do that in the PR where we work on the new `local`.
I wanted to have remove `sig: ''` here because `signing-api` assumes the signature is signed if it sees the empty string.